pkg/gen/nodejs-templates/yaml.ts.mustache

```diff
@@ -64,7 +82,15 @@ import * as outputApi from "../types/output";
             }
 
             for (const file of files) {
-                const cf = new ConfigFile(file, {file: file, transformations: config.transformations}, opts);
+                const cf = new ConfigFile(
+                    file,
+                    {
+                        file: file,
```

Comment:
Is there a reason why `file` is set here twice, both in the `ConfigFile` 1st param, and in the map object in this line?

Reply:
Yeah, for this case we're autonaming the resources after the filename. We might be able to consolidate it, but it's out of scope for this change. If we ever have time, it might be worth taking a cleanup pass through this file.